### ISSUE TITLE
System nameserver value is overriden by profile nameserver value

### ISSUE THREAD

**Issue by cknapp** [cobbler/cobbler#1468]
Description: The nameserver value set by the systems host entry is overriden by the value set by the profile of host.

System info:
```
[root@cobbler kickstarts]# uname -r
3.10.0-229.el7.x86_64
[root@cobbler kickstarts]# rpm -qa | grep cob
cobbler-2.6.7-1.el7.noarch
cobbler-web-2.6.7-1.el7.noarch
[root@cobbler kickstarts]# cat /etc/centos-release
CentOS Linux release 7.1.1503 (Core) 
```

Repro:
1. Create a profile (profile_A) and set the nameserver (Networking tab) to 1.1.1.1 
2. Create a machine entry (machine_1) and set the profile to profile_A
3. Set the nameservers value in the machine entry to 8.8.8.8
4. Click Save
5. Click View Kickstart

Expected: Nameserver is 8.8.8.8
Actual: Nameserver is 1.1.1.1

**Comment by jmaas** [comment]
```
[root@cobbler cobbler]# cobbler profile list
   centos-7.0-x86_64
   debian-8.0.0-gtk-x86_64
   debian-8.0.0-x86_64
   debian-8.0.0-xen-x86_64
   ubuntu-14.04.2-x86_64
   ubuntu-15.04-x86_64

[root@cobbler cobbler]# cobbler profile edit --name=centos-7.0-x86_64 --name-servers=1.1.1.1
[root@cobbler cobbler]# cobbler profile dumpvars --name=centos-7.0-x86_64 |grep -i name_servers
<snip>
name_servers : ['1.1.1.1']
<snip>
[root@cobbler cobbler]# cobbler system add --name=xxx --profile=centos-7.0-x86_64
[root@cobbler cobbler]# cobbler system dumpvars --name=xxx |grep name_servers
<snip>
name_servers : ['1.1.1.1']
<snip>
[root@cobbler cobbler]# cobbler system edit --name=xxx --name-servers=8.8.8.8
[root@cobbler cobbler]# cobbler system dumpvars --name=xxx |grep name_servers
<snip>
name_servers : ['1.1.1.1', '8.8.8.8']
<snip>
[root@cobbler cobbler]# 
```

**Comment by jmaas** [comment]
```
[root@cobbler cobbler]# cat /etc/redhat-release 
CentOS Linux release 7.1.1503 (Core) 
[root@cobbler cobbler]# cobbler version
Cobbler 2.6.8
  source: f155386, Wed May 6 20:38:36 2015 +0200
  build time: Wed May  6 21:07:17 2015
[root@cobbler cobbler]# cat /etc/redhat-release 
CentOS Linux release 7.1.1503 (Core) 
```

**Comment by jmaas** [comment]
perhaps this was fixed in 2.6.8, but i cant imagine it was... 

can you reproduce using the cli commands i used ??

**Comment by jmaas** [comment]
Since i can't reproduce this i'm closing this issue. If this still is an issue, please reproduce using the commands i used, or otherwise provide enough information so i can verify. In that case i'm happy to re-open this issue.
